app/api/tickets.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+from app.api.helpers.jwt import jwt_required
+from flask_rest_jsonapi import ResourceDetail, ResourceList, ResourceRelationship
+from marshmallow_jsonapi.flask import Schema, Relationship
+from marshmallow_jsonapi import fields
+from app.models import db
+from app.models.ticket import Ticket
+from app.models.event import Event
+
+
+class TicketSchema(Schema):
+
+    class Meta:
+        type_ = 'ticket'
+        self_view = 'v1.ticket_detail'
+        self_view_kwargs = {'id': '<id>'}
+
+    id = fields.Str(dump_only=True)
+    name = fields.Str()
+    description = fields.Str()
+    type = fields.Str()
+    price = fields.Float()
+    quantity = fields.Integer()
+    price = db.Column(db.Float)
```

Comment:
@niranjan94 this has been fixed #3706.
I'll open and issue and fix all the comment reviews along with the tax api?